### PR TITLE
Revert #554 and require fenicsadapter 2.2.0 to fix subcycling

### LINKED ISSUE #573: `perpendicular-flap/solid-fenics`: Regression
There seems to be a regression in `perpendicular-flap/solid-fenics` when looking at the tip displacement:

image

https://github.com/precice/tutorials/pull/554 is a candidate. Maybe the issue is related to checkpointing and how the values are updated here. If I remember correctly, we only checked the results with the fake fluid solver. Here, no checkpointing is needed.

*edit:* I used `fenicsprecice==v2.1.0` for these results. However, with `fenicsprecice==v2.2.0rc1` it is also possible to reproduce the results.

### PR DESCRIPTION
**Pull request by NiklasVin**
This resolves #573 by reverting #554.
For more information, refer to #573.

### CODE REVIEW

**Review by BenjaminRodenberg — COMMENTED** [review]
I'm testing the changes now. Without subcycling, `sold-dealii` and `solid-fenics` are in good agreement. When I perform subcycling as described in https://github.com/precice/fenics-adapter/pull/172 I also get the errors we have seen there.

As a solution we have to do the following:
* https://github.com/precice/fenics-adapter/pull/184
* Require `fenicsprecice>=2.2` for this tutorial

I will take care of these steps.

I'm currently performing some tests with the pre-release `v2.2.0rc1` and subcycling and the results look good.

File: perpendicular-flap/solid-fenics/requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-fenicsprecice~=2.0
+fenicsprecice==2.2.0rc1
```
Comment (BenjaminRodenberg) [inline perpendicular-flap/solid-fenics/requirements.txt]:
```suggestion
fenicsprecice~=2.2
```
As soon as https://github.com/precice/fenics-adapter/pull/184 is merged and the new release is available.

**Comment by BenjaminRodenberg** [comment]
@MakisH can you run the systemtests using this branch? This should work now if the systemtest pick up fenicsprecice 2.2.0 correctly.

**Comment by MakisH** [comment]
Hm, actually, no: we have a limitation at the system tests at the moment, that we cannot check out commits from forks (opened an issue: #580). If the commit was in this branch, the command to trigger the system tests would have been:

```
gh workflow run run_testsuite_manual.yml -f suites=release_test -f build_args="PRECICE_REF:v3.1.2,OPENFOAM_ADAPTER_REF:v1.3.1,PYTHON_BINDINGS_REF:v3.1.2,FENICS_ADAPTER_REF:v2.2.0,SU2_VERSION:7.5.1,SU2_ADAPTER_REF:64d4aff,TUTORIALS_REF:53fea5c" --ref=develop
```

(yes this can be simplified similarly to other repositories, opened an issue #581)

I suggest merging if you know that this is something ready to be merged and then re-check on develop.

**Comment by BenjaminRodenberg** [comment]
I did a final round of testing similar to the ones in #573:

image

Results look good and we are back on track :+1: I will merge this PR now.

**Comment by BenjaminRodenberg** [comment]
@MakisH systemtests should work now.

**Pull request merged**